app/templates/gulpfile.js

```diff
@@ -25,13 +25,23 @@ gulp.task('styles', function () {<% if (includeSass) { %>
     .pipe(reload({stream: true}));
 });
 
-gulp.task('jshint', function () {
-  return gulp.src('app/scripts/**/*.js')
-    .pipe(reload({stream: true, once: true}))
-    .pipe($.jshint())
-    .pipe($.jshint.reporter('jshint-stylish'))
-    .pipe($.if(!browserSync.active, $.jshint.reporter('fail')));
-});
+/**
+ * @param {String|Array} files to provide to `gulp.src`
+ * @return {Function} callback for `gulp.task` et al
+ */
+function jshint (files) {
+  return function(){
```

Comment:
We should fix `function jshint (files)` too so it reads `function jshint(files) {`.